scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3937,9 +3937,19 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       accept[Equals]
       Defn.GivenAlias(mods, sigName, sigTparams, sigUparamss, decltpe, exprMaybeIndented())
     } else {
```

Comment:
comments section for this method:
` // GivenDef          ::=  [GivenSig] [‘_’ ‘<:’] Type ‘=’ Expr`
are no longer relevant, probably best to remove

Reply:
Updated!